content/tutorials/pulumi-deployments-click-to-deploy/index.md

```diff
@@ -0,0 +1,120 @@
+
+## Setup a new Pulumi Deployment
+
+First sign in to Pulumi Cloud via the [Pulumi sign in page](https://app.pulumi.com) or by running `pulumi login` via the Pulumi CLI.
+
+## Install and configure the GitHub App
+
+Next, install and configure the [Pulumi GitHub App](/docs/using-pulumi/continuous-delivery/github-app/) to enable Pulumi Deployment's New Project Wizard and push-to-deploy functionality. Follow the GitHub app [installation instructions](/docs/pulumi-cloud/deployments/reference/#github-app-installation) to ensure your Pulumi organization is set up to work seamlessly with Pulumi Deployments.
+
+## Create a New Project with the New Project Wizard
+
+To create a new Pulumi project, commit its code, and deploy it entirely from your browser, you will use the [New Project Wizard](/docs/pulumi-cloud/developer-portals/new-project-wizard/). The wizard allows you to start from one of Pulumi’s public templates, your [Organization Templates](/docs/pulumi-cloud/developer-portals/templates), or even generate a custom project using the cloud resources and your preferred language of choice with [Pulumi AI](link).
+
+First, navigate to the **New Project** tab in the Pulumi Cloud console. Choose your source template from Pulumi, then you can optionally choose your cloud and language to filter the results or search all templates.
+
+![Animation of how to create a new project in Pulumi Cloud](./pulumi-create-new-project.gif)
+
+{{% notes type="info" %}}
+If your Pulumi administrator has configured [custom templates](/docs/pulumi-cloud/developer-portals/templates) for your organization, you will be able to choose from your organization's custom templates from your own image source.
+If you select "Use a template" but your organization doesn't have custom templates, you can choose one of Pulumi's public templates to get started.
+{{% /notes %}}
+
+Type `aws-python` or `aws-typescript` and click **next**.
+
+### Project details
+
+Provide a name for the project, along with an optional description and the stack name which will be set in the resulting `Pulumi.yaml` file and the name of `Pulumi.<stack>.yaml`.
+
+### Configuration
+
+Provide any values required for the configuration declared in the [template](/docs/pulumi-cloud/developer-portals/templates), in this example, you can set the **aws:region** you would like to deploy into.
+
+These values populate the `config` section in the `Pulumi.<stack>.yaml` file, specifying the necessary settings for your project.
+
+### Environment
+
+If you've configured [environments](/docs/pulumi-cloud/esc) with your organization, you can specify one to use with the resulting stack. Environments enable teams to manage hierarchical collections of configuration and secrets without needing to re-specify all of them during project creation.
```

Comment:
The word "hierarchical" feels superfluous here, given the tutorial doesn't use any hierarchical config. 

```suggestion
If you've configured [environments](/docs/pulumi-cloud/esc) with your organization, you can specify one to use with the resulting stack. Environments enable teams to define and share collections of configuration across multiple Pulumi projects and stacks.
```